modules/lambda/README.md

```diff
@@ -2,18 +2,31 @@ This module creates multiple lambdas that can be invoked by an API Gateway. Each
 
 # Example
 
-```
+```hcl
 module "lambda" { 
-  source = git::https://github.com/betabandido/terraformmodules.git//modules/lambda"
+  source      = git::https://github.com/betabandido/terraformmodules.git//modules/lambda"
   lambda_file = "lambda.zip"
-  function_names_and_handlers = {
-    LambdaModuleTest1 = "package.say_hello"
-    LambdaModuleTest2 = "package.say_goodbye"
+
+  functions = {
+    LambdaModuleTest1 = {
+      handler = "package.say_hello"
+    }
+    LambdaModuleTest2 = {
+      handler = "package.say_goodbye"
+    }
   }
-  source_arn = "arn:aws:execute-api:${var.region}:${var.accountId}:${aws_api_gateway_rest_api.api.id}/*/GET/*/*"
-  policy = "${data.template_file.example_policy.rendered}"
-  prefix = "MyLambdas"
-  runtime = "python3.6"
+
+  variables = {
```

Comment:
Maybe is it better to call them `env_variables` or something similar. In Terraform, it's true that the map is called just `variables`, but they're inside another map called `environment`, so it's clear. In this case, we may not know the type of variables we're talking about.